dask-apps/cudf_merge_mpi.py

```diff
@@ -0,0 +1,594 @@
+"""
```

Comment:
Could we move this somewhere else? Maybe [dask-examples]( https://github.com/dask/dask-examples ) would make sense?

Reply:
Yes we can do that. The only reason to put them here was to have sample code for the MPI backend. 

My colleague @ZackWRyan will help with this PR.

Reply:
I'll remove both of these examples from this PR, we can make a separate PR on dask-examples to go along with this one later.